pkg/cmd/server/origin/master.go

```diff
@@ -148,24 +148,42 @@ func (c *MasterConfig) BuildClients() {
 	c.osClient = osclient
 }
 
+// KubeClient returns the kubernetes client object
 func (c *MasterConfig) KubeClient() *kclient.Client {
 	return c.kubeClient
 }
+
+// OSClient returns the openshift client object
+func (c *MasterConfig) OSClient() *osclient.Client {
```

Comment:
No, you can't do this.  You have to call a method that says what component is using the client.  There is no "generic use of the client", because eventually everyone stops using this and starts using the others.  Create a method for the name of the component that is acting.